core/src/chiplets/bitwise.rs

```diff
@@ -15,21 +15,15 @@ pub const OP_CYCLE_LEN: usize = 8;
 // --- OPERATION SELECTORS ------------------------------------------------------------------------
 
 /// Specifies a bitwise AND operation.
-pub const BITWISE_AND: Selectors = [Felt::ZERO, Felt::ZERO];
+pub const BITWISE_AND: Selectors = [Felt::ZERO];
 /// Unique label for the bitwise AND operation. Computed as 1 more than the binary composition of
-/// the chiplet and operation selectors [1, 0, 0, 0].
+/// the chiplet and operation selectors [1, 0, 0].
 pub const BITWISE_AND_LABEL: Felt = Felt::new(2);
 
-/// Specifies a bitwise OR operation.
-pub const BITWISE_OR: Selectors = [Felt::ZERO, Felt::ONE];
-/// Unique label for the bitwise OR operation. Computed as 1 more than the binary composition of the
-/// chiplet and operation selectors [1, 0, 0, 1].
-pub const BITWISE_OR_LABEL: Felt = Felt::new(10);
-
 /// Specifies a bitwise XOR operation.
-pub const BITWISE_XOR: Selectors = [Felt::ONE, Felt::ZERO];
+pub const BITWISE_XOR: Selectors = [Felt::ONE];
```

Comment:
Does this need to be an array? Or can we get away with making it a simple `Felt`?

Reply:
Changed it to felt and did the corresponding refactoring.